app/src/main/java/jayp/bge/playboikotlin/robbie/fragments/RobbieFragmentsActivity.kt

```diff
@@ -0,0 +1,34 @@
+package jayp.bge.playboikotlin.robbie.fragments
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.AppBarConfiguration
+import androidx.navigation.ui.setupWithNavController
+import jayp.bge.playboikotlin.R
+import jayp.bge.playboikotlin.databinding.ActivityRobbieFragmentsBinding
+
+class RobbieFragmentsActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityRobbieFragmentsBinding
+    private lateinit var navController: NavController
```

Comment:
we should cut this lateinit bullshit out. :/

Reply:
why should we cut this